manager/integration/tests/conftest.py

```diff
@@ -173,3 +177,38 @@ def pytest_collection_modifyitems(config, items):
         for item in items:
             if "upgrade" in item.keywords:
                 item.add_marker(skip_upgrade)
+
+
+def pytest_exception_interact(call, report):
+
+    # Get case name
+    case_name = str(report).split()[1]
+    case_name = case_name.split('/')[1].replace('\'', '')
+
+    # Use API gen support bundle
```

Comment:
Create a function for support bundle download in common.py, then reuse it here instead.